static/templates/user_profile_modal.handlebars

```diff
@@ -37,18 +37,20 @@
         <hr>
         <div id="content">
             {{#each profile_data}}
-            <div data-type="{{this.type}}" class="field-section">
-                <div class="name">{{this.name}}</div>
-                {{#if this.is_user_field}}
-                <div class="pill-container not-editable" data-field-id="{{this.id}}">
-                    <div class="input" contenteditable="false" style="display: none;"></div>
+                {{#if this.value}}
```

Comment:
How does the refactor to add the outer `if this_value` relate to the rest (and why is it correct)?  Maybe split that out as a preparatory commit, and then the remaining commit will be just about user type fields.